openlane/steps/klayout.py

```diff
@@ -346,7 +346,7 @@ class DRC(KLayoutStep):
         ),
         Variable(
             "KLAYOUT_DRC_OPTIONS",
-            Optional[Dict[str, Union[bool, int]]],
+            [Dict[str, Union[bool, int]]],
```

Comment:
shouldn't be an array (for when you wake up)

Reply:
Why array?

Reply:
Enclosed by `[]`